providers/base/units/camera/test-plan.pxu

```diff
@@ -68,6 +69,7 @@ _description: Camera tests After Suspend (automated)
 include:
  after-suspend-camera/detect                                     certification-status=blocker
  after-suspend-camera/multiple-resolution-images_.*              certification-status=blocker
+ camera/camera-quality_*                                         certification-status=non-blocker
```

Comment:
Prefix our job with `after-suspend-` here. This version of your checkbox camera job is automatically instantiated if the flag `after-suspend-` is used

Reply:
I have added the after-suspend flag